Tests/function/tests/Update-VSTeamUserEntitlement.Tests.ps1

```diff
@@ -16,15 +16,33 @@ Describe "VSTeamUserEntitlement" -Tag 'VSTeamUserEntitlement' {
          Mock _callAPI -ParameterFilter { $Method -eq 'Patch' }
 
          # Get-VSTeamUserEntitlement by email
-         Mock _callAPI { Open-SampleFile 'Get-VSTeamUserEntitlement.json' }
+         Mock _callAPI { Write-Host $args;Open-SampleFile 'Get-VSTeamUserEntitlement.json' }
 
          # Get-VSTeamUserEntitlement by id
-         Mock _callAPI { Open-SampleFile 'Get-VSTeamUserEntitlement-Id.json' } -ParameterFilter {
+         Mock _callAPI { Write-Host $args;Open-SampleFile 'Get-VSTeamUserEntitlement-Id.json' } -ParameterFilter {
             $id -eq '00000000-0000-0000-0000-000000000000'
          }
       }
 
-      It 'by email should update a user' {
+      It 'by email without license should update a user' {
+         Update-VSTeamUserEntitlement `
+            -LicensingSource msdn `
+            -MSDNLicenseType enterprise `
+            -Email 'dlbm3@test.com' `
+            -Force
+
+         Should -Invoke _callAPI -Exactly -Times 1 -Scope It -ParameterFilter {
```

Comment:
There are three values that are being passed up to the API in each of these requests.  The only thing the function does is determine if there's a new value to apply or just pass back the current value.

This test should assert that the submitted update object has the expected license type since that isn't changing per the update call.

Reply:
I really tried to understand what you are saying. But I don't get what I should test in your opinion. The licenses are asserted, arent't they?

the cdmdlet does not have return so I can't test the correct return values. I will go ahead with the bug fix, but maybe you could still comment on this and I can improve the test later.